tests/windows.rs

```diff
@@ -95,6 +95,76 @@ fn windows_iterator_3d() {
     );
 }
 
+/// Test that verifies the `Windows` iterator panics when stride has an axis equal to zero.
+#[test]
+#[should_panic]
+fn windows_iterator_stride_axis_zero() {
+    let a = Array::from_iter(10..37).into_shape((3, 3, 3)).unwrap();
+    a.windows_with_stride(Dim((2, 2, 2)), Dim((0,2,2)));
```

Comment:
I'm aware that the original maintainer used `Dim` here, but it's useless. It just makes the code longer and harder to read. You can do like you did in the test below: directly use the tuple `(2, 2, 2)`. Please change in all the new tests.